libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -853,6 +855,44 @@ public CharSequence getContent(CharSequence originalContent) throws EditorFragme
         });
     }
 
+
+    @Override
+    public void showContentInfo() throws EditorFragmentNotAddedException {
+        if (!isAdded()) {
+            throw new EditorFragmentNotAddedException();
+        }
+
+        getGutenbergContainerFragment().triggerGetContentInfo(new OnContentInfoReceivedListener() {
+            @Override
+            public void onContentInfoFailed() {
+                ToastUtils.showToast(getActivity(), R.string.toast_content_info_failed);
```

Comment:
Both the `onContentInfoReceived` callback and this `onContentInfoFailed` callback get invoked from the background thread that is created in `WPAndroidGlueCode::triggerGetContentInfo`, so we need to switch to the main thread here like we do in `onContentInfoReceived` in order to avoid a crash when we try to create the toast, right?

Reply:
Indeed, that worked in some early tests I did but I refactored the code later and these code paths were forgotten :(

Good catch! Fixed with 9fbbcb821c8cdbdc17aabb07b5398636735a53f0.